src/runner.ts

```diff
@@ -140,5 +148,12 @@ export async function run(fn: Runner, args: string[], options: DetectOptions = {
     return
   }
 
-  await ezspawn(command, { stdio: 'inherit', cwd })
+  const [xCommand, ...xArguments] = command.split(' ')
```

Comment:
I don't think it's save to use split, as you might have edge cases like `run "foo bar"`, we would either need to complete change how we compose the commands (use array instead of string), or we need tinyexec to support parsing string commands

Reply:
This should be easy since we have the original arguments, I'll check tmr.